src/components/Videos.vue

```diff
@@ -121,6 +122,211 @@ export default {
+				fr: 'French',
+				fy: 'Western Frisian',
+				ga: 'Irish',
+				gd: 'Gaelic',
+				gl: 'Galician',
+				gn: 'Guarani',
+				gu: 'Gujarati',
+				gv: 'Manx',
+				ha: 'Hausa',
+				he: 'Hebrew',
+				hi: 'Hindi',
+				ho: 'Hiri Motu',
+				hr: 'Croatian',
+				ht: 'Haitian',
+				hu: 'Hungarian',
+				hy: 'Armenian',
+				hz: 'Herero',
+				ia: 'Interlingua',
+				id: 'Indonesian',
+				ie: 'Interlingue',
+				ig: 'Igbo',
+				ii: 'Sichuan Yi',
+				ik: 'Inupiaq',
+				io: 'Ido',
+				is: 'Icelandic',
+				it: 'Italian',
+				iu: 'Inuktitut',
+				ja: 'Japanese',
+				jv: 'Javanese',
+				ka: 'Georgian',
+				kg: 'Kongo',
+				ki: 'Kikuyu',
+				kj: 'Kuanyama',
+				kk: 'Kazakh',
+				kl: 'Kalaallisut',
+				km: 'Central Khmer',
+				kn: 'Kannada',
+				ko: 'Korean',
+				kr: 'Kanuri',
+				ks: 'Kashmiri',
+				ku: 'Kurdish',
+				kv: 'Komi',
+				kw: 'Cornish',
+				ky: 'Kirghiz',
+				la: 'Latin',
+				lb: 'Luxembourgish',
+				lg: 'Ganda',
+				li: 'Limburgan',
+				ln: 'Lingala',
+				lo: 'Lao',
+				lt: 'Lithuanian',
+				lu: 'Luba-Katanga',
+				lv: 'Latvian',
+				mg: 'Malagasy',
+				mh: 'Marshallese',
+				mi: 'Maori',
+				mk: 'Macedonian',
+				ml: 'Malayalam',
+				mn: 'Mongolian',
+				mr: 'Marathi',
+				ms: 'Malay',
+				mt: 'Maltese',
+				my: 'Burmese',
+				na: 'Nauru',
+				nb: 'Norwegian Bokmål',
+				nd: 'Ndebele, North',
+				ne: 'Nepali',
+				ng: 'Ndonga',
+				nl: 'Dutch',
+				nn: 'Norwegian Nynorsk',
+				no: 'Norwegian',
+				nr: 'Ndebele, South',
+				nv: 'Navajo',
+				ny: 'Nyanja',
+				oc: 'Occitan',
+				oj: 'Ojibwa',
+				om: 'Oromo',
+				or: 'Oriya',
+				os: 'Ossetian',
+				pa: 'Panjabi',
+				pi: 'Pali',
+				pl: 'Polish',
+				ps: 'Pushto',
+				pt: 'Portuguese',
+				qu: 'Quechua',
+				rm: 'Romansh',
+				rn: 'Rundi',
+				ro: 'Romanian',
+				ru: 'Russian',
+				rw: 'Kinyarwanda',
+				sa: 'Sanskrit',
+				sc: 'Sardinian',
+				sd: 'Sindhi',
+				se: 'Northern Sami',
+				sg: 'Sango',
+				si: 'Sinhala',
+				sk: 'Slovak',
+				sl: 'Slovenian',
+				sm: 'Samoan',
+				sn: 'Shona',
+				so: 'Somali',
+				sq: 'Albanian',
+				sr: 'Serbian',
+				ss: 'Swati',
+				st: 'Sotho',
+				su: 'Sundanese',
+				sv: 'Swedish',
+				sw: 'Swahili',
+				ta: 'Tamil',
+				te: 'Telugu',
+				tg: 'Tajik',
+				th: 'Thai',
+				ti: 'Tigrinya',
+				tk: 'Turkmen',
+				tl: 'Tagalog',
+				tn: 'Tswana',
+				to: 'Tonga',
+				tr: 'Turkish',
+				ts: 'Tsonga',
+				tt: 'Tatar',
+				tw: 'Twi',
+				ty: 'Tahitian',
+				ug: 'Uighur',
+				uk: 'Ukrainian',
+				ur: 'Urdu',
+				uz: 'Uzbek',
+				ve: 'Venda',
+				vi: 'Vietnamese',
+				vo: 'Volapük',
+				wa: 'Walloon',
+				wo: 'Wolof',
+				xh: 'Xhosa',
+				yi: 'Yiddish',
+				yo: 'Yoruba',
+				za: 'Zhuang',
+				zh: 'Chinese',
+				zu: 'Zulu',
+			}
+			const langs = this.$attrs.caplangs
+			let tracks = ''
+			// Create caption tracks for the HTML5 player
+			// E.g.: movie.mkv --> movke.en.vtt
+			for (let i = 0; i < langs.length; ++i) {
+				const lang = langs[i]
```

Comment:
```suggestion
			for (let lang of langs) {
```

But it might be even better to use `v-for` instead.